packages/protocol/contracts/verifiers/libs/LibProofHash.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "../../L1/ITaikoL1.sol";
+
+/// @title LibProofHash
+/// @notice A library for handling hashing the so-called public input hash, used by sgx and zk
+/// proofs.
+/// @custom:security-contact security@taiko.xyz
+library LibProofHash {
```

Comment:
Since the hash is the "public input" hash, LibPublicInputHash may be a more approriate name